### PR TITLE
Create copy-files-with-scp-and-rsync/

### PR DESCRIPTION
**Pull request by BadCrumpany**

### CODE REVIEW

File: content/cloud-servers/copy-files-with-scp-and-rsync.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+title: 'Copy files with SCP and RSYNC'
+type: article
+created_date: '2020-06-18'
+created_by: Jose Quezada
+last_modified_date:
+last_modified_by:
+product: Cloud Servers
+product_url: cloud-servers
+---
+
```
Comment (chri2547) [inline content/cloud-servers/copy-files-with-scp-and-rsync.md:13]: This article guides you in transferring a file to a remote server.

File: content/cloud-servers/copy-files-with-scp-and-rsync.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+This guide is intended to provide guidance when you try to transfer a file from/to a remote server.
+
+# Copying files/directories in a remote way to/from your server
+
+There are two different tools to transfer files remotely in an easy way, SSH and RSYNC. The difference between them resides in that SCP uses SSH to copy files or directories indicated by user. RSYNC on the other way the first time is used copy all files/ directories and the further times only copy a delta (files modified, new files or directories), it does not copy all files/directories again.
```
Comment (chri2547) [inline content/cloud-servers/copy-files-with-scp-and-rsync.md:18]: You can use SSH or RSYNC to transfer a file.

Comment (chri2547) [inline content/cloud-servers/copy-files-with-scp-and-rsync.md:18]: Why are you mention SCP in the 2nd sentence?

RSYNC initially copies all files, and then during subsequent operations, RSYNC only copies changed files.

File: content/cloud-servers/copy-files-with-scp-and-rsync.md (hunk 1 above)
Comment (catlook) [reply content/cloud-servers/copy-files-with-scp-and-rsync.md:13]: cloud-servers is where server content goes in the how-to. there is no remote server section.

**Comment by cyrichardson** [comment]
On it, @chri2547.

**Comment by chri2547** [comment]
@BadCrumpany , all set on this end. We're ready to have you give final approval and then merge.

**Comment by catlook** [comment]
build me

**Comment by rackernexus** [comment]
Build 2028 was successful. Staging content [Preview!](https://support-revs.staging.deconst.org/build-8043bb66e/how-to/). valid for 14 days

**Pull request merged**